### PR TITLE
synfig etl 1.5.2

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- https://github.com/synfig/synfig/issues/3382
- https://github.com/Homebrew/homebrew-core/pull/177929
- https://github.com/synfig/synfig/issues/3371

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  sed "s/PACKAGE/ETL/g;" < ../ETL/etl_profile_.h > ../ETL/etl_profile.h
  make[2]: Nothing to be done for `install-exec-am'.
   .././config/install-sh -c -d '/opt/homebrew/Cellar/etl/1.5.2/include/ETL/ETL'
  /bin/sh: .././config/install-sh: /usr/bin/sh: bad interpreter: No such file or directory
```

**Comment by chenrui333** [comment]
- https://github.com/synfig/synfig/issues/3398

File: Formula/s/synfig.rb [hunk 1]
```diff
@@ -29,14 +21,12 @@ class Synfig < Formula
...
-  end
-
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "intltool" => :build
-  depends_on "pkg-config" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => [:build, :test]
+
   depends_on "boost"
```
Comment (ice0) [inline Formula/s/synfig.rb:30]: `boost` is no longer needed

Comment (chenrui333) [reply Formula/s/synfig.rb]: Thanks! 👍

File: Formula/s/synfig.rb [hunk 2]
```diff
@@ -55,20 +44,38 @@ class Synfig < Formula
...
-      system "./bootstrap.sh"
-    end
-    system "./configure", *std_configure_args,
-                          "--disable-silent-rules",
+    # missing install-sh in the tarball, and re-generate configure script
+    # upstream bug report, https://github.com/synfig/synfig/issues/3398
+    system "autoreconf", "--force", "--install", "--verbose"
+
+    system "./configure", "--disable-silent-rules",
                           "--with-boost=#{Formula["boost"].opt_prefix}",
```
Comment (morevnaproject) [inline Formula/s/synfig.rb:76]: Boost is not required anymore :)

Comment (chenrui333) [reply Formula/s/synfig.rb]: 👍 :)

**Comment by MikeMcQuaid** [comment]
> some sourceforge artifact audit failure

Please open a PR or issue for this before merging this PR, thanks!

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by chenrui333** [comment]
> > some sourceforge artifact audit failure
> 
> Please open a PR or issue for this before merging this PR, thanks!

you mean like handling such issue on the brew side?

**Comment by MikeMcQuaid** [comment]
> you mean like handling such issue on the brew side?

@chenrui333 I don't know what the issue is here and what needs changed where. An issue explaining that or starting discussion would be good, thanks!

**Comment by chenrui333** [comment]
Oh, I see. (we kind of like always run into the sourceforge artfact check failures from time to time, let me see what I can do)

**Comment by chenrui333** [comment]
logged an issue and gonna proceed with the merge.

**Pull request merged**